docs/range-sources.adoc

```diff
@@ -28,9 +28,9 @@ It can be positive or negative:
 @ParameterizedTest
 @DoubleRangeSource(from = -0.1, to = -10, step = -0.1)
 void howColdIsIt(double d) {
-	System.out.println(d + " degrees Celsius is cold");
-	System.out.println(d + " degrees Fahrenheit is REALY cold");
-	System.out.println(d + " degrees Kelvin is too cold to be true");
+	System.out.println(d + " °C is cold");
```

Comment:
Oh, non-ASCII chars in code snippets? 😬

Reply:
(Extended) ASCII-Code 248 :p http://www.asciitable.com/

I havn't come up with the short limit of 60 chars where about 40 are for intenden and `System.out.println(..);` 🤷‍♂️

Reply:
Every day's a school say. 👍